leukeleu_django_gdpr/checks.py

```diff
@@ -0,0 +1,27 @@
+from django.core.checks import Warning, register
+
+from leukeleu_django_gdpr.management.commands.gdpr import get_pii_stats
+
+
+class Tags:
+    confidentiality = "confidentiality"
+
+
+W001 = Warning(
+    "You have one or more model field(s) without a PII classification.",
+    hint="Update gpdr.yml and mark all field with either pii: true or pii: false.",
+    id="leukeleu_django_gdpr.W001",
+)
+
+
+@register(Tags.confidentiality, deploy=True)
```

Comment:
Maybe this should be a non-deploy check (to make sure we fix this before merging/deploying)